pages/docs/changelog.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: 'docs'
+title: 'Changelog'
+metaTitle: 'Changelog'
+metaDescription: 'History der Changelogs'
+---
+  
+<p className="lead">
+  Erfahre mehr über die aktuelle Version, Bugs und neue Funktionen der Plattform.
+</p>
+
+Die Plattform versteht sich als Prototyp, wobei der Quellcode komplett Open Source ist und auf Github in den [Repositories der Technologiestiftung Berlin](https://github.com/technologiestiftung?q=iot&type=&language=&sort=) gefunden werden kann. 
```

Comment:
Link to this repo?

Reply:
was not sure which Repo to link cuz at the end we have one repo for the API and one for the frontend. What do you consider as mother repo?

Reply:
changed quyer to stadtpuls --> https://github.com/technologiestiftung?q=stadtpuls&type=&language=&sort=

Reply:
Nevermind my initial comment. It makes sense this way (once the query is changed to `stadtpuls`)